### PR TITLE
use @fastify/error instead of http-errors

### PR DESCRIPTION
**Pull request by Uzlopak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,15 +29,15 @@
...
+    "fastify-plugin": "^3.0.0"
   },
   "devDependencies": {
-    "@types/node": "^18.0.0",
-    "fastify": "^4.0.0-rc.2",
     "@fastify/cookie": "^7.0.0",
     "@fastify/secure-session": "^5.0.0",
     "@fastify/session": "^9.0.0",
+    "@types/node": "^18.0.0",
+    "fastify": "^4.0.0-rc.2",
```
Comment (kibertoad) [inline package.json:40]: why not final version?

Comment (Uzlopak) [reply package.json:40]: npm i reordered the entries. Any suggestion?

Comment (kibertoad) [reply package.json:40]: let's bump to latest while we are at it :(

Comment (Uzlopak) [reply package.json:40]:
```suggestion
    "fastify": "^4.2.0",
```

**Review by kibertoad — APPROVED** [review]

**Comment by Uzlopak** [comment]
@kibertoad 
Is 403 Forbidden actually the right code. I think it should be 401 Unauthorized. Should i open an issue to discuss this?

**Comment by kibertoad** [comment]
Good point. And yes, let's discuss in separate issue.

**Comment by Uzlopak** [comment]
@kibertoad 
I did a short Google search, i guess 403 is the correct Code

**Pull request merged**